### PR TITLE
feat: set editor theme by predefined theme/storybook theme or fallback to light theme

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/5677797274

### CODE REVIEW

File: src/components/Editor/Editor.tsx [hunk 1]
```diff
@@ -17,13 +19,20 @@ type EditorComponent = React.ForwardRefExoticComponent<
...
 const Editor: EditorComponent = forwardRef(
   ({ type, code, fontSize, extensions, onChange }, ref) => {
+    const { theme: storybookTheme } = useStorybookState();
+    const { editorTheme: addonTheme } = useParameter<PlaygroundParameters>(
+      ADDON_ID_FOR_PARAMETERS,
+      DEFAULT_ADDON_PARAMETERS
+    );
     const placeholder = `Insert your ${type.toUpperCase()} code here`;
+    const theme: EditorColor =
+      addonTheme || storybookTheme.base === "dark" ? "dark" : "light";
```
Comment (talkor) [inline src/components/Editor/Editor.tsx]:
```suggestion
    const theme: EditorColor =
      addonTheme || storybookTheme.base;
```

File: src/consts/parameter-consts.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { PlaygroundParameters } from "@/types";
 
 export const DEFAULT_ADDON_PARAMETERS: PlaygroundParameters = {
-  playgroundStoryId: "playground",
+  storyId: "playground",
   components: {},
+  editorTheme: null,
```
Comment (talkor) [inline src/consts/parameter-consts.ts:6]: Why null? Override the default?

File: src/types.ts [hunk 3]
```diff
@@ -25,3 +26,5 @@ export interface PlaygroundState {
 export type Code = { jsx: string; css: string };
 
 export type Tab = "jsx" | "css";
+
+export type EditorColor = "light" | "dark";
```
Comment (talkor) [inline src/types.ts]:
```suggestion
export type EditorTheme = "light" | "dark";
```

File: src/components/Editor/Editor.tsx (hunk 1 above)
Comment (YossiSaadi) [reply src/components/Editor/Editor.tsx]: It can also be "normal", that's the reaosn for it 😥

**Review by talkor — APPROVED** [review]
LGTM

File: src/components/Editor/Editor.tsx (hunk 1 above)
Comment (talkor) [reply src/components/Editor/Editor.tsx]: Doesn't seem normal to me

File: src/consts/parameter-consts.ts (hunk 2 above)
Comment (YossiSaadi) [reply src/consts/parameter-consts.ts:6]: If it would have default light it would never query storybook's theme, enforcing the user to always declare it

**Pull request merged**